### PR TITLE
Nolana fix tests

### PR DESCRIPTION
**Pull request by IhorBohdan**

### CODE REVIEW

File: cypress/e2e/finance/funds/add-expense-class-to-budget.cy.js [hunk 1]
```diff
@@ -6,9 +6,7 @@ import testType from '../../../support/dictionary/testTypes';
 import devTeams from '../../../support/dictionary/devTeams';
 import NewExpenceClass from '../../../support/fragments/settings/finance/newExpenseClass';
 import SettingsFinance from '../../../support/fragments/settings/finance/settingsFinance';
-import FiscalYears from '../../../support/fragments/finance/fiscalYears/fiscalYears';
 import TopMenu from '../../../support/fragments/topMenu';
-import Ledgers from '../../../support/fragments/finance/ledgers/ledgers';
 import NewFiscalYear from '../../../support/fragments/finance/fiscalYears/newFiscalYear';
```
Comment (penfriend) [inline cypress/e2e/finance/funds/add-expense-class-to-budget.cy.js:10]: 'NewFiscalYear' is defined but never used. Allowed unused vars must match /React/u

File: cypress/e2e/finance/funds/add-expense-class-to-budget.cy.js [hunk 2]
```diff
@@ -6,9 +6,7 @@ import testType from '../../../support/dictionary/testTypes';
...
 import NewLedger from '../../../support/fragments/finance/ledgers/newLedger';
```
Comment (penfriend) [inline cypress/e2e/finance/funds/add-expense-class-to-budget.cy.js:11]: 'NewLedger' is defined but never used. Allowed unused vars must match /React/u

File: cypress/e2e/finance/funds/test-acquisition-unit-restrictions-for-funds.cy.js [hunk 3]
```diff
@@ -91,6 +91,7 @@ describe('ui-finance: Funds', () => {
     cy.login(user.username, user.password, { path:TopMenu.fundPath, waiter: Funds.waitLoading });
     FinanceHelp.searchByName(defaultfund.name);
     Funds.checkZeroSearchResultsHeader();
+    cy.logout();
```
Comment (penfriend) [inline cypress/e2e/finance/funds/test-acquisition-unit-restrictions-for-funds.cy.js:94]: why do you need a logout? isn't it obsolete?

File: cypress/e2e/finance/groups/add-multiple-funds-to-group.cy.js [hunk 4]
```diff
@@ -92,6 +92,7 @@ describe('ui-finance: Groups', () => {
     Groups.deleteGroupViaApi(defaultGroup.id);
     Ledgers.deleteledgerViaApi(defaultLedger.id);
     FiscalYears.deleteFiscalYearViaApi(defaultFiscalYear.id);
+    cy.wait(2000);
```
Comment (penfriend) [inline cypress/e2e/finance/groups/add-multiple-funds-to-group.cy.js:95]: why do you need this waiter?

File: cypress/support/fragments/settings/finance/settingsFinance.js [hunk 5]
```diff
@@ -42,7 +42,7 @@ export default {
     cy.do(MultiColumnListCell({ content: expenseClass.name }).perform(
       element => {
         const rowNumber = element.parentElement.parentElement.getAttribute('data-row-index');
-        const createdByAdmin = `${DateTools.getFormattedDateWithSlashes({ date: new Date() })} by ADMINISTRATOR, DIKU`;
```
Comment (penfriend) [inline cypress/support/fragments/settings/finance/settingsFinance.js:45]: Please use more universal approach. Take the Last name[, First name] from the profileDropdown

Comment (penfriend) [reply cypress/support/fragments/settings/finance/settingsFinance.js:45]: image

File: cypress/support/fragments/settings/acquisitionUnits/acquisitionUnits.js [hunk 6]
```diff
@@ -61,7 +61,7 @@ export default {
   assignAdmin: () => {
     cy.do([
       findUserButton.click(),
-      userSearchModal.find(searchTextField).fillIn('diku'),
+      userSearchModal.find(searchTextField).fillIn('folio-aqa'),
```
Comment (penfriend) [inline cypress/support/fragments/settings/acquisitionUnits/acquisitionUnits.js:64]: Please use more universal approach. Take the Last name[, First name] from the profileDropdown

File: cypress/support/fragments/orders/orders.js [hunk 7]
```diff
@@ -34,9 +34,9 @@ const searchButton = Button('Search');
 const newButton = Button('New');
 const saveAndClose = Button('Save & close');
 const orderDetailsAccordionId = 'purchaseOrder';
-const createdByAdmin = 'ADMINISTRATOR, DIKU ';
+const createdByAdmin = 'folio-aqa  ';
 const searchField = SearchField({ id: 'input-record-search' });
-const admin = 'administrator';
+const admin = 'folio-aqa';
```
Comment (penfriend) [inline cypress/support/fragments/orders/orders.js:39]: Please use more universal approach. Take the Last name[, First name] from the profileDropdown

File: cypress/support/fragments/orders/orders.js [hunk 8]
```diff
@@ -34,9 +34,9 @@ const searchButton = Button('Search');
 const newButton = Button('New');
 const saveAndClose = Button('Save & close');
 const orderDetailsAccordionId = 'purchaseOrder';
-const createdByAdmin = 'ADMINISTRATOR, DIKU ';
+const createdByAdmin = 'folio-aqa  ';
```
Comment (penfriend) [inline cypress/support/fragments/orders/orders.js:37]: Please use more universal approach. Take the Last name[, First name] from the profileDropdown

File: cypress/support/fragments/invoices/settingsInvoices.js [hunk 9]
```diff
@@ -49,7 +49,7 @@ export default {
     cy.do(MultiColumnListCell({ content: batchGroup.name }).perform(
       element => {
         const rowNumber = element.parentElement.parentElement.getAttribute('data-row-index');
-        const createdByAdmin = `${DateTools.getFormattedDateWithSlashes({ date: new Date() })} by ADMINISTRATOR, DIKU`;
+        const createdByAdmin = `${DateTools.getFormattedDateWithSlashes({ date: new Date() })} by folio-aqa`;
```
Comment (penfriend) [inline cypress/support/fragments/invoices/settingsInvoices.js:52]: Please use more universal approach. Take the Last name[, First name] from the profileDropdown

**Review by penfriend — CHANGES_REQUESTED** [review]

**Review by TetianaParanich — APPROVED** [review]

**Review by penfriend — APPROVED** [review]

File: cypress/e2e/finance/groups/add-multiple-funds-to-group.cy.js (hunk 4 above)
Comment (IhorBohdan) [reply cypress/e2e/finance/groups/add-multiple-funds-to-group.cy.js:96]: Answer in TO DO

File: cypress/e2e/finance/funds/test-acquisition-unit-restrictions-for-funds.cy.js (hunk 3 above)
Comment (IhorBohdan) [reply cypress/e2e/finance/funds/test-acquisition-unit-restrictions-for-funds.cy.js]: Fixed

**Pull request merged**